bitcoin/src/util/psbt/mod.rs

```diff
@@ -1686,7 +1685,6 @@ mod tests {
         assert_eq!(psbt1, psbt2);
     }
 
-    #[cfg(feature = "rand")]
     fn gen_keys() -> (PrivateKey, PublicKey, Secp256k1<All>) {
```

Comment:
WTF, this shouldn't work. I suspect we have something seriously wrong, maybe we're enabling `rand-std` via other means.

Reply:
I thought the same but "rand-std" is enabled in the dev dependencies.

Reply:
Oh, maybe that's wrong because we can't properly check that the features work correctly?

Reply:
Fixed! Thanks for prodding me to fix it correctly.